### PR TITLE
Refactor specs to avoid using sample and insist

### PR DESCRIPTION
**Pull request by purbon**

### CODE REVIEW

**Comment by purbon** [comment]
Can you take a quick look @guyboertje ? thanks a lot.

File: spec/filters/csv_spec.rb [hunk 1]
```diff
@@ -1,175 +1,160 @@
...
-    CONFIG
-
-    sample "big,bird;sesame street" do
-      insist { subject["column1"] } == "big,bird"
-      insist { subject["column2"] } == "sesame street"
+    let(:doc) { "big,bird;sesame street" }
+
+    let(:config) do
+      { "separator" => ";" }
+    end
```
Comment (guyboertje) [inline spec/filters/csv_spec.rb]: Consider updating the config hash you created in line 8 e.g.
In the `it block` add `config.update("separator" => ";")`

Only if you want to. :smile:

**Comment by guyboertje** [comment]
LGTM

File: spec/filters/csv_spec.rb (hunk 1 above)
Comment (purbon) [inline spec/filters/csv_spec.rb]: :+1: I tend to prefer having inheritance like this, doing update might be confusing, don't you think?

**Pull request closed without merging**